fpmsyncd/fpmsyncd.cpp

```diff
@@ -16,7 +16,9 @@ using namespace swss;
  * Default warm-restart timer interval for routing-stack app. To be used only if
  * no explicit value has been defined in configuration.
  */
-const uint32_t DEFAULT_ROUTING_RESTART_INTERVAL = 120;
+const uint32_t DEFAULT_ROUTING_RESTART_INTERVAL = 600;
```

Comment:
What is the purpose of increasing this timeout?

Reply:
This timer is now similar to the fallback timer in fdbsyncd.  This will take effect only to avoid indefinite wait on orchagent getting reconciled.  The default reconcile timer is still 120 seconds as defined by the other macro 
const uint32_t DEFAULT_ROUTING_RECON_CHECK_INTERVAL = 120;

Reply:
Same as above, we should well define these two timers: DEFAULT_ROUTING_RESTART_INTERVAL and DEFAULT_ROUTING_RECON_CHECK_INTERVAL in document/code-comments.

Reply:
Sure will add comment in the code